### ISSUE TITLE
Allow .jl in the package names on badges

### ISSUE THREAD

**Issue by imciner2** [GenieFramework/PkgVizBoard#1]
For packages that wrap a library in other languages, the badge can be useful to put on the main library's readme to show its popularity. Unfortunately, right now it would just show the library name and not have any way of saying it is the Julia binding. It would be nice to be able to put the `.jl` after the package name to show it is the Julia binding (so then the badge shows `PackageName.jl downloads`). This could probably be done by allowing the badge URL to have the `.jl` in the name given to it, and then just stripping it out when needed in the processing to get the statistics (this way would be less disruptive to the current uses).

**Comment by essenciary** [comment]
@imciner2 Are you thinking about JLL files?

**Comment by imciner2** [comment]
No, I am thinking about the package that would provide the glue to the Julia language, the JLL only provides the library. The usecase I have in mind is the OSQP package. There are wrappers for multiple languages for the C library, and the upstream readme in the repo has badges counting the Python downloads (saying "Conda downloads" and "Pypi downloads"), but if I try to add the badge for the Julia package that wraps the library (it is OSQP.jl), the badge just says "OSQP Downloads" and makes no indication it is for the Julia wrapper and can be construed as being a more generic download count. I was hoping to have the badge say "OSQP.jl Downloads", so then it is clear it is for the Julia wrapper.

Note I am not suggesting that all packages get the .jl in the suffix, but that it be allowed to be specified when making the badge using the URL.
